android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TranslationsStateReducer.kt

```diff
@@ -190,6 +191,16 @@ internal object TranslationsStateReducer {
                 )
             }
 
+        is TranslationsAction.RemoveNeverTranslateSiteAction -> {
+            var neverTranslateSites = state.findTab(action.tabId)?.translationsState?.neverTranslateSites
```

Comment:
I think this and below maybe can be `val`.

Reply:
updated to `val`s